policy/approval/approve.go

```diff
@@ -40,6 +40,7 @@ type Options struct {
 	AllowContributor   bool `yaml:"allow_contributor"`
 	InvalidateOnPush   bool `yaml:"invalidate_on_push"`
 	IgnoreUpdateMerges bool `yaml:"ignore_update_merges"`
+	RequestReview      bool `yaml:"request_review"`
```

Comment:
I think we might want to make this a struct from the beginning to allow easy addition of further options. For instance, if we need a concept of priorities or if we want to adjust the random selection processes, those options would probably go here.